packages/simulator/web/routes.js

```diff
@@ -146,6 +146,10 @@ function register(io) {
           socket.emit('bookings', bookings)
         }
       })
+    experiment.buses.pipe(map(cleanCars)).subscribe((e) => {
```

Comment:
This shouldn't be needed? Look at changing boolean on line 87 instead which would have an effect in emitOnlyCars-pipeline below?